### PR TITLE
Add p1 Issue query and separate issues by repo

### PR DESCRIPTION
**Pull request by donnie-msft**
- Separate "all" vars by repo.
- Add queries for Priority1 Open issues by repo
- Write a JSON file with those P1 issues for export.

### CODE REVIEW

File: GithubIssueTagger/IssueUtilities.cs [hunk 1]
```diff
@@ -40,6 +40,23 @@ public static async Task<IEnumerable<Issue>> GetAllIssues(GitHubClient client, s
...
+
+            var queryLabels = new string[] { "priority:1" };
+
+            var request = new SearchIssuesRequest()
+            {
+                Repos = nugetRepos,
+                State = ItemState.Open,
+                Labels = queryLabels
+            };
+            var issuesForMilestone = await client.Search.SearchIssues(request);
```
Comment (donnie-msft) [inline GithubIssueTagger/IssueUtilities.cs:56]: SearchIssues is a workaround for the other api we were using.
Had weird JSON deserialization issues, and found this bug: https://github.com/octokit/octokit.net/issues/1762

File: GithubIssueTagger/Program.cs [hunk 2]
```diff
@@ -9,8 +11,10 @@ namespace GithubIssueTagger
     public class Program
     {
         private static IList<Issue> _unprocessedIssues;
-        private static IReadOnlyList<Label> _allLabels;
-        private static IEnumerable<Issue> _allIssues;
+        private static IReadOnlyList<Label> _allHomeLabels;
```
Comment (nkolev92) [inline GithubIssueTagger/Program.cs:14]: I don't want to add more to work for, but these statics make me cringe :) 

It'd be neater if you were using return values/passed collections etc.

**Review by nkolev92 — APPROVED** [review]

File: GithubIssueTagger/Program.cs (hunk 2 above)
Comment (donnie-msft) [reply GithubIssueTagger/Program.cs:14]: This change really has nothing to do with the statics, but I simply had to split them into 2 collections. Someone can separately PR refactoring if we think that's needed.

Comment (nkolev92) [reply GithubIssueTagger/Program.cs:14]: Yeah, more of a general comment. 

I might do the refactoring opportunistically if you don't mind.

**Pull request merged**